### PR TITLE
Adding documentation about rocksdb

### PR DESCRIPTION
**Pull request by senior7515**
/cc @RBlafford

### CODE REVIEW

File: source/introduction.rst [hunk 1]
```diff
@@ -55,9 +55,10 @@ guarantees that errant processes don't cause other processes to crash.
 .. rubric:: 3. Performance
 
 Concord's communication layer is built from the ground up in C++14, utilizing
-libevent_, fbthrift_ and RocksDB_. Message delivery between operators in Concord
-achieves low latency through efficient, non-blocking network IO similar to that
-of nginx.
+libevent_ for efficient evented I/O, thrift_ for the communication layer, and
```
Comment (bigs) [inline source/introduction.rst]: utilizing thrift - backed by libevent for efficient evented I/O - for the communication layer

File: source/introduction.rst [hunk 2]
```diff
@@ -55,9 +55,10 @@ guarantees that errant processes don't cause other processes to crash.
...
+RocksDB_ for backpressure. Message delivery between operators in
```
Comment (bigs) [inline source/introduction.rst]: RocksDB to manage backpressure.

**Comment by RBlafford** [comment]
LGTM

**Comment by senior7515** [comment]
Thanks! guys.

**Pull request merged**